notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -822,6 +843,16 @@ private void checkToRemoveDomainInUrl(Page note) {
     }
   }
 
+  private boolean canCreateNotes(String authenticatedUser, Space space) throws WikiException {
+    return space != null &&
```

Comment:
Not sure that this is the good condition for edit permissions please check again